lighthouse/lighthouserc.js

```diff
@@ -79,7 +98,7 @@ module.exports = {
       assertions: {
         'categories:performance': [
           'warn',
-          { aggregationMethod: 'optimistic', minScore: 0.7 },
+          { aggregationMethod: 'optimistic', minScore: 0.4 },
```

Comment:
Performance scores are very poor, but do not cause the Lighthouse build to fail... see https://eu-west-1.console.aws.amazon.com/codesuite/codebuild/657504540040/projects/simorgh-cd-envtest/build/simorgh-cd-envtest%3A05824ac4-1b1f-4ec3-ad23-34b805e12c22/?region=eu-west-1 and scroll up to see the performance failures: 
![image](https://github.com/bbc/simorgh/assets/58214768/325ff75a-71d5-4790-a234-25e6c6c6795d)

I plan to update the Simorgh Infra buildspec to make sure lighthouse tests fail if any of these metrics fail